agent/app/agent_capability_unix.go

```diff
@@ -232,3 +235,14 @@ func (agent *ecsAgent) getTaskENIPluginVersionAttribute() (*ecs.Attribute, error
 func defaultIsPlatformExecSupported() (bool, error) {
 	return true, nil
 }
+
+func defaultIsGuardDutySupported() (bool, error) {
+	isEnabled, err := pathExists(config.GuardDutyServiceFilePath, false)
+	if err != nil {
+		logger.Warn("GuardDuty Capability: Cannot find GuardDuty service file path. GuardDuty capabilities will "+
```

Comment:
I don't think this should be a warning, this would log a warning for every customer that doesn't have the guardduty RPM installed right?

Reply:
logger.Debug seems more apt

Reply:
Modified this.

Reply:
Since I updated this change in the latest commit,Resolving this request.